### PR TITLE
Super tafani member

### PR DESCRIPTION
**Pull request by Mahmoudmohamedalbanhawi**

### CODE REVIEW

File: Super Tafani Member/index.html [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                            </svg>
+                    </span>
+                    <input type="text" class="form-control ms-0 pt-3 pb-2 lightgrey-bg border-0" placeholder="Search By keyword" aria-label="Username" aria-describedby="basic-addon1">
+                  </div>
+             <div class=" w-28">
+                <select class="form-select border-0 py-3 lightgrey-bg" aria-label="Disabled select example" disabled>
```
Comment (markfady) [inline Super Tafani Member/index.html]: Please make sure from the color of arrow icon in `select` it should be `#8E9AA0` like figma

File: Super Tafani Member/index.html [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                        </svg>
+                        
+                        <p class="coral">Show details</p>
+                    </td>
```
Comment (markfady) [inline Super Tafani Member/index.html:179]: Please remove this `</td>` as it is a syntax error.

File: Super Tafani Member/index.html [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+                            </svg>
+                    </span>
+                    <input type="text" class="form-control ms-0 pt-3 pb-2 lightgrey-bg border-0" placeholder="Search By keyword" aria-label="Username" aria-describedby="basic-addon1">
+                  </div>
+             <div class=" w-28">
+                <select class="form-select border-0 py-3 lightgrey-bg" aria-label="Disabled select example" disabled>
+                    <option selected>Status</option>
```
Comment (markfady) [inline Super Tafani Member/index.html:99]: Remove `disabled `attribute to show these options

File: Super Tafani Member/index.html [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+                <div class="w-28">
+                    <select class="form-select py-3 border-0 " aria-label="Disabled select example" disabled>
+                        <option selected>Date</option>
+                        <option value="1">One</option>
+                        <option value="2">Two</option>
+                        <option value="3">Three</option>
+                      </select>
+                </div>
+                <div class="w-28 mt-2">
+                    <div class="pill" role="group" aria-label="Basic mixed styles example">
```
Comment (markfady) [inline Super Tafani Member/index.html:114]: Please check the space between this `pill `and the `select` element; it needs to be **increased** slightly

File: Super Tafani Member/index.html [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+    
+      <div class="container-fluid">
+        <div class="row ">
+            <aside class="col-2 side px-0 primary-background-bg  min-vh-100" >
+                <ul>
+                 <li class="text-white nav-item mt-60 px-30" ><a href="#Dashboard" class="nav-link">Dashboard</a></li>
+                 <li class="text-white nav-item mt-22 px-30"><a href="#My Customers" class="nav-link">My Customers</a></li>
+                 <li class="text-white nav-item mt-34 px-30"><a href="#Community" class="nav-link">Community</a></li>
+                 <li class="text-white nav-item mt-28 px-30"><a href="#Leaderboard" class="nav-link">Leaderboard</a></li>
+                 <li class="text-white nav-item mt-28 px-30"><a href="#Products" class="nav-link">Products</a></li>
```
Comment (markfady) [inline Super Tafani Member/index.html]: Please fix this typo it is Products & Services not Products

File: Super Tafani Member/index.html [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+                        </svg>
+                        
+                        <p class="coral">Show details</p>
+                    </td>
+                    </td>
+                  </tr>
+                  <tr>
+                    <td scope="row">999961</td>
+                    <td >Larry the Bird</td>
```
Comment (markfady) [inline Super Tafani Member/index.html]: Please fix this typo it must be **Question**

File: Super Tafani Member/index.html (hunk 1 above)
Comment (Mahmoudmohamedalbanhawi) [reply Super Tafani Member/index.html]: done

File: Super Tafani Member/index.html (hunk 2 above)
Comment (Mahmoudmohamedalbanhawi) [reply Super Tafani Member/index.html:179]: done

File: Super Tafani Member/index.html (hunk 3 above)
Comment (Mahmoudmohamedalbanhawi) [reply Super Tafani Member/index.html:99]: finish

File: Super Tafani Member/index.html (hunk 4 above)
Comment (Mahmoudmohamedalbanhawi) [reply Super Tafani Member/index.html:114]: done

File: Super Tafani Member/index.html (hunk 5 above)
Comment (Mahmoudmohamedalbanhawi) [reply Super Tafani Member/index.html]: done

File: Super Tafani Member/index.html (hunk 6 above)
Comment (Mahmoudmohamedalbanhawi) [reply Super Tafani Member/index.html]: done

**Comment by markfady** [comment]
Thanks Mahmoud for your efforts. Keep it up! 👏

**Pull request merged**